Documentation/Signoffs/clturney42-Output-signoff.md

```diff
@@ -0,0 +1,168 @@
+
+### 7) Speaker shall have an input power rating maximum of 16 watts |Origin: Amplifier specifications and power insurance|
+Due to the amplifier being able to supply a continuous power of 16 W [1], an appropriate speaker must be selected so that the amplifier has enough power to drive it. A good rule of thumb is to have the amplifier used have an output power appropriately higher than the speakers power rating. With this in mind, the speaker that is used should have a power rating of 15 watts or less. This ensures that the amplifier will be able to consistently drive the speaker with enough power, even if there are slight fluctuations in the system. 
+
+### 8) Output system shall extend a maximum of 0.5 meters from the wall |Origin: Ethics and system delay|
+To satisfy the ethical constraints of movement impedence and reduce delay as much as possible in the system, the speaker will extend a maximum of 0.5 meters from the wall. This distance will likely be less but this constraint is set to make sure that the speaker is never in the way of individuals within the room. This distance constraint is also necessary due to the delay associated with the speed of sound and distance of the speaker from the sound source. The speaker cannot have too much delay else the anti-noise will not be in sync with outside noise.
+  
+### 9) Output system shall be positioned close enough to wall to reduce delay enough to eliminate echo in sound |Origin: Device specifications and goal of system|
+  
+An ideal noise cancellation device would have no delay however, that is impossible to do when the signal is being processed and traveling through different hardware. With this in mind, the delay of the device needs to be set so that there is as minimal as possible disparity between the inputted signal and outputted noise. This can be verified and changed based on the distance the speaker is positioned from the wall.
+  
+### 10) Speaker shall be mounted, and will not impede movement or damage the classroom
+
+The speaker will be fitted into an enclosure that matches the dimensions of the speaker. This enclosure will then be mounted to the wall in an optimal position for noise cancellation. To fit this constraint, the position in which the speaker is placed will not impede movement or damage the classroom. This can be verified by confirming the placement of the speaker does not block the classroom or damage the walls or windows within the classroom.
+
+### 11) Speaker shall have a sound pressure level and efficiency that is sufficient to permeate the room |Origin: Derived Contraint from room|
```

Comment:
I think that the question shouldn't be if the system can permeate the room, but how loud of a sound can it cancel.

Reply:
I believe I went over this in the analysis for constraint 11, I have changed the wording of the constraint but the analysis goes over how the SPL of the speaker will maintain a level that is higher than that of construction noises, so it should be able to cancel it.